packages/design-system/src/components/popup/constants.js

```diff
@@ -51,3 +56,21 @@ export const RTL_PLACEMENT = {
   [PLACEMENT.LEFT_START]: PLACEMENT.RIGHT_START,
   [PLACEMENT.LEFT_END]: PLACEMENT.RIGHT_END,
 };
+
+export const PopupContainer = styled.div.attrs(
+  ({ $offset: { x, y, width }, fillWidth, transforms, invisible, zIndex }) => ({
+    style: {
+      transform: `translate(${x}px, ${y}px) ${transforms}`,
+      ...(fillWidth ? { width: `${width}px` } : {}),
+      ...(invisible ? { visibility: 'hidden' } : {}),
+      zIndex,
+    },
+  })
+)`
+  /*! @noflip */
+  left: 0px;
+  top: 0px;
+  position: fixed;
+  ${({ noOverFlow }) => (noOverFlow ? '' : `overflow-y: auto;`)};
```

Comment:
nit (I know this was copy/paste) 
```suggestion
  ${({ noOverFlow }) => (noOverFlow ? '' : 'overflow-y: auto;')};
```

Reply:
Nice catch. Thanks.